TestRailAutomationTest/Test/LoginTest.cs

```diff
@@ -0,0 +1,19 @@
+using FluentAssertions;
+using NUnit.Framework;
+using TestRailAutomationTest.Page;
+
+namespace TestRailAutomationTest.Test;
+
+public class LoginPageTest : BaseTest
+{
+    [Test]
+    public void testIsOneCanLogIn()
```

Comment:
Naming 
I would suggest this style :

**_What_Condition_ExpectedBehavior_**

`Login_WithValidCredentials_ShouldBeSuccsessful`